matrix/common/aws/dynamo_handler.py

```diff
@@ -161,6 +151,44 @@ def increment_table_field(self, table: DynamoTable, request_id: str, field_enum:
         start_value, end_value = self._increment_field(dynamo_table, key_dict, field_enum, increment_size)
         return start_value, end_value
 
+    def set_table_field_with_value(self, table: DynamoTable, request_id: str, field_enum: TableField, field_value: str):
+        """
+        Set value in dynamo table
+        Args:
+            table: DynamoTable enum
+            request_id: request id key in table
+            field_enum: field enum to increment
+            field_value: Value to set for field
+        """
+        dynamo_table = self._get_dynamo_table_resource_from_enum(table)
+        key_dict = {"RequestId": request_id}
+        self._set_field(dynamo_table, key_dict, field_enum, field_value)
+
+    def _set_field(self, table, key_dict: dict, field_enum: TableField, field_value: str):
+        """
+        Set a value in a dynamo table.
+        Args:
+          table: boto3 resource for a dynamodb table
+          key_dict: Dict for the key in the table
+          field_enum: Name of the field to increment
+          field_value: Value to set for field
+        """
+        field_enum_value = field_enum.value
+        while True:
+            try:
+                table.update_item(
+                    Key=key_dict,
+                    UpdateExpression=f"SET {field_enum_value} = :n",
+                    ExpressionAttributeValues={":n": field_value}
+                )
+                break
+            except botocore.exceptions.ClientError as exc:
+                if exc.response['Error']['Code'] == "ConditionalCheckFailedException":
+                    pass
```

Comment:
Could this cause an infinite loop? i.e. when would this case run?

Reply:
yea I'm deleting this code. This was only useful for when multiple lambdas were incrementing completed executions (ie copy and paste leftover)